policy-test/tests/accepted_http_route_status.rs

```diff
@@ -0,0 +1,380 @@
+        assert_eq!(route_status.parent_ref.name, server.name_unchecked());
+
+        // Check status is accepted with a status of 'True'
+        let cond = route_status
+            .conditions
+            .iter()
+            .find(|cond| cond.type_ == "Accepted")
+            .expect("must have at least one 'Accepted' condition");
+        assert_eq!(cond.status, "True");
+        assert_eq!(cond.reason, "Accepted")
+    })
+    .await;
+}
+
+#[tokio::test(flavor = "current_thread")]
+async fn inbound_multiple_parents() {
+    with_temp_ns(|client, ns| async move {
+        // Exercise accepted test with a valid, and an invalid parent reference
+        let srv_refs = vec![
+            k8s::policy::httproute::ParentReference {
+                group: Some("policy.linkerd.io".to_string()),
+                kind: Some("Server".to_string()),
+                namespace: Some(ns.clone()),
+                name: "test-valid-server".to_string(),
+                section_name: None,
+                port: None,
+            },
+            k8s::policy::httproute::ParentReference {
+                group: Some("policy.linkerd.io".to_string()),
+                kind: Some("Server".to_string()),
+                namespace: Some(ns.clone()),
+                name: "test-invalid-server".to_string(),
+                section_name: None,
+                port: None,
+            },
+        ];
+
+        // Create only one of the parents
+        let server = k8s::policy::Server {
+            metadata: k8s::ObjectMeta {
+                namespace: Some(ns.to_string()),
+                name: Some("test-valid-server".to_string()),
+                ..Default::default()
+            },
+            spec: k8s::policy::ServerSpec {
+                pod_selector: k8s::labels::Selector::from_iter(Some(("app", "test-valid-server"))),
+                port: k8s::policy::server::Port::Name("http".to_string()),
+                proxy_protocol: Some(k8s::policy::server::ProxyProtocol::Http1),
+            },
+        };
+        let _server = create(&client, server).await;
+
+        // Create a route that references both parents.
+        let _route = create(
+            &client,
+            mk_inbound_route(&ns, "test-multiple-parents-route", Some(srv_refs)),
+        )
+        .await;
+        // Wait until route is updated with a status
+        let parent_status = await_route_status(&client, &ns, "test-multiple-parents-route")
+            .await
+            .expect("condition timed out")
+            .parents;
+
+        // Find status for invalid parent and extract the condition
+        let invalid_cond = parent_status
+            .iter()
+            .find(|route_status| route_status.parent_ref.name == "test-invalid-server")
+            .expect("must have a route status set for invalid server")
+            .conditions
+            .clone()
+            .into_iter()
+            .find(|cond| cond.type_ == "Accepted")
```

Comment:
nit, take it or leave it: it looks like we do this enough that we could _probably_ add a `find_condition` helper (or something) that takes an `impl IntoIter<Item = &RouteStatus>` and a `&str` for the parent ref name and returns an `Option<Condition>`?